gobblin-yarn/src/main/java/org/apache/gobblin/yarn/GobblinYarnAppLauncher.java

```diff
@@ -173,6 +174,8 @@ public class GobblinYarnAppLauncher {
 
   private static final String GOBBLIN_YARN_APPLICATION_TYPE = "GOBBLIN_YARN";
 
+  private static final String APPLICATION_TAGS_KEY = "hadoop-inject.mapreduce.job.tags";
```

Comment:
to anticipate confusion, this may be worth a code comment, that even though this is named for an MR prop:
https://hadoop.apache.org/docs/stable/hadoop-mapreduce-client/hadoop-mapreduce-client-core/mapred-default.xml

that it's essentially morphed into a general-purpose pass-through for Azkaban to provide context about its execution

the `hadoop-inject.` prefix is documented here - https://azkaban.readthedocs.io/en/latest/jobTypes.html#id4

Reply:
I suppose you could even link here - https://github.com/azkaban/azkaban/blob/6db750049f6fdf7842e18b8d533a3b736429bdf4/az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/AbstractHadoopJavaProcessJob.java#L96